dali_tf_plugin/dali_dataset_op.cc

```diff
@@ -881,6 +983,7 @@ REGISTER_OP("DALIDataset")
     .Output("handle: variant")
     .Attr("input_names: list(string)")    // must match the input_datasets
     .Attr("input_layouts: list(string)")  // must match the input_datasets
+    .Attr("input_batched: list(int)")  // must match the input_datasets, use int instead of bool
```

Comment:
why is that (int instead of bool)?

Reply:
I grepped for `list(bool)` in TF, and found like free uses, only in test. So I prefer to not use `vector<bool>`